### PR TITLE
Update IdCategories.tsx

### PR DESCRIPTION
**Pull request by alexgtang**
## Link to Issue

https://github.com/keepid/keepid_client/issues/466

## Description of changes

Changed wording in enum

### CODE REVIEW

**Review by mkhagan8 — APPROVED** [review]
With my limited programming knowledge, it looks good to me.

**Comment by steffen12** [comment]
I think we should do "'Drivers License / Photo ID' with a space surrounding the /

File: src/components/Documents/IdCategories.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 export enum IdCategories {
-  DriversLicense = 'Drivers License',
+  DriversLicense = 'Drivers License/Photo ID',
```
Comment (steffen12) [inline src/components/Documents/IdCategories.tsx:2]: Change

File: src/components/Documents/IdCategories.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
   BirthCertificate = 'Birth Certificate',
   SocialSecurityCard = 'Social Security Card',
   VaccineCard = 'Vaccine Card',
-  MedicaidCard = 'Medicaid Card',
+  MedicaidCard = 'Medical Insurance Card',
```
Comment (steffen12) [inline src/components/Documents/IdCategories.tsx:6]: Good

**Review by steffen12 — CHANGES_REQUESTED** [review]

**Review by steffen12 — APPROVED** [review]
Looks Good

**Pull request merged**